examples/pyqt5_qml/basicqml/main.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+function JSManager() {
+    return Qt.createQmlObject("import JSManager 1.0; JSManager {}",
+                              appWindow, "JSManager");
+}
+
+function onLoad(){
+    console.log('inside');
+    var jsManager = new JSManager();
+
+    // Populate text
+    jsManager.text.connect(function(text){
+      console.log(text);
```

Comment:
It's not a problem, I was just amused that we're now packaging interpreters for both Python and Javascript in one installer. :-)